### PR TITLE
[v6r14] Better error reporting (affecting jobs application status)

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

File: Core/Security/File.py [hunk 1]
```diff
@@ -1,37 +1,43 @@
...
+      return S_ERROR( "Failed to create temporary file" )
     fileName = proxyLocation
   try:
     fd = open( fileName, 'w' )
     fd.write( proxyContents )
     fd.close()
-  except Exception, e:
-    return S_ERROR( "Cannot write to file %s :%s" % ( fileName, str( e ) ) )
+  except Exception as e:
+    gLogger.error( "Cannot write to file", " %s: %s" % ( fileName, str( e ) ) )
```
Comment (chaen) [inline Core/Security/File.py]: %s already calls str on the object

File: Core/Security/File.py [hunk 2]
```diff
@@ -1,37 +1,43 @@
...
+    return S_ERROR( "Cannot write to file" )
   try:
     os.chmod( fileName, stat.S_IRUSR | stat.S_IWUSR )
-  except Exception, e:
-    return S_ERROR( "Cannot set permissions to file %s :%s" % ( fileName, str( e ) ) )
+  except Exception as e:
+    gLogger.error( "Cannot set permissions to file", "%s: %s" % ( fileName, str( e ) ) )
```
Comment (chaen) [inline Core/Security/File.py]: same here

**Comment by chaen** [comment]
I guess the changes in JobScheduling.py have to be updated with respect to https://github.com/DIRACGrid/DIRAC/pull/2604

**Comment by fstagni** [comment]
Yep, updated here.

**Comment by fstagni** [comment]
I will re-do this branch for v6r15, adding DError

**Pull request closed without merging**